amundsen_application/static/js/components/TableDetail/ColumnListItem/ColumnType/styles.scss

```diff
@@ -31,6 +32,8 @@ $modal-dialog-width: 418px;
     height: calc(
       #{$modal-dialog-height - $modal-header-height - $spacer-3}
     );
+    font-family: $font-family-spacemono;
+    font-size: 12px;
```

Comment:
Would make sense to have a $code-font-size variable or similar?

Reply:
Sure.